modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsUniqueDate.testcase

```diff
@@ -22,5 +22,63 @@ definition {
 	}
 
 
+	@description = "LPS-135398 - Verify unique values can be enforced for text fields"
+	@priority = 5
+	test CanPreventDuplicateText {
+		property custom.properties = "feature.flag.LPS-135398=true";
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a text field") {
```

Comment:
Use the `task ("Given a custom object that contains a text field with unique values only')`